src/main/kotlin/no/nav/familie/ef/personhendelse/inntekt/InntektOppgaveService.kt

```diff
@@ -84,6 +92,31 @@ class InntektOppgaveService(
         return kandidater.size
     }
 
+    fun opprettOppgaveForPersonSomHarFyltTjueFemOgHarArbeidsavklaringspenger(
+        skalOppretteOppgave: Boolean,
+    ): Int {
+        val inntektsendringForBrukereMedArbeidsavklaringspenger = inntektsendringerRepository.hentInntektsendringerForPersonMedArbeidsavklaringspenger()
+        val startDato = YearMonth.now().minusMonths(1).atDay(6)
+        val sluttDato = LocalDate.now()
+
+        val kandidater =
+            inntektsendringForBrukereMedArbeidsavklaringspenger.mapNotNull { endring ->
+                val person = pdlClient.hentPerson(endring.personIdent)
+                val foedselsdato = person.foedselsdato.first().foedselsdato
```

Comment:
fun List<Fødselsdato>.gjeldende(): Fødselsdato = this.find { !it.metadata.historisk } ?: this.first()